### PR TITLE
Header avatar with menu and user avatar

### PR DESCRIPTION
**Pull request by davideastmond**
- Update language dropdown to use muiSelect
- Headerbar avatar with 'logout' menu
- User avatar (for event page, user profile) that can be customized

### CODE REVIEW

File: travel-event-host-app/src/components/header/Header.tsx [hunk 1]
```diff
@@ -33,32 +40,83 @@ export default function Header() {
...
+                        backgroundColor: theme.palette.primary.secondaryColorDarkBlack,
+                        color: theme.palette.primary.thirdColorIceLight,
+                        borderTop: 'none',
+                        '& .MuiButtonBase-root:hover': {
+                          backgroundColor: theme.palette.primary.primaryColorDarkBlue,
+                        },
+                      },
+                    },
+                  },
+                }}
```
Comment (davideastmond) [inline travel-event-host-app/src/components/header/Header.tsx]: This is all styling to get the language menu to look like it did before.

File: travel-event-host-app/src/components/header/Header.tsx [hunk 2]
```diff
@@ -1,15 +1,22 @@
...
-import Avatar from '@mui/material/Avatar';
-import CircularProgress, { circularProgressClasses } from '@mui/material/CircularProgress';
-import Language from '@/lib/language';
 export default function Header() {
   const [lang, setLang] = useState<Language>(Language.En);
   const [status, setStatus] = useState('unauthenticated');
   const [navMenuIsOpen, setnavMenuIsOpen] = useState(false);
   const userName = 'Angelo';
+
+  const handleLanguageOptionChanged = (event: SelectChangeEvent) => {
```
Comment (LifeAsDev) [inline travel-event-host-app/src/components/header/Header.tsx:16]: why is better have a handle event if just set the state?

File: travel-event-host-app/src/components/header/Header.tsx [hunk 3]
```diff
@@ -33,32 +40,83 @@ export default function Header() {
...
+                    fontSize: '11px',
+                  },
+                },
+              }}
+            >
+              <Select
+                labelId='selectLanguageDropdown'
+                id='selectLanguageDropdown'
+                value={lang}
+                onChange={handleLanguageOptionChanged}
```
Comment (LifeAsDev) [inline travel-event-host-app/src/components/header/Header.tsx]: so instead handleLanguageOptionChanged
```suggestion
              onChange={(event) => setLang(event.target.value as Language)}

```

File: travel-event-host-app/src/components/header/Header.tsx (hunk 2 above)
Comment (davideastmond) [reply travel-event-host-app/src/components/header/Header.tsx:16]: Yes, no need. I'll update

**Review by LifeAsDev — APPROVED** [review]

**Pull request merged**